.github/ISSUE_TEMPLATE/infrastructure.md

```diff
@@ -0,0 +1,22 @@
+---
+name: I want to file a bug or feature request to the infrastructure
```

Comment:
If this is public on our site, I bet a lot of people won't know what we mean by "infrastructure" without more context.  Maybe instead of "to the infrastructure", "about Flutter's CI infrastructure"?

Reply:
Sounds good, reworded.